lib/rules/property-no-unknown/__tests__/index.js

```diff
@@ -360,3 +360,64 @@ testRule({
 		},
 	],
 });
+
+testRule({
+	ruleName,
+	config: [true, { ignoreAtRules: ['media', '/^my-/'] }],
+
+	accept: [
+		{
+			code: 'a { @media { foo: 1 } }',
+		},
+		{
+			code: 'a { b { @media { foo: 1 }}}',
+		},
+		{
+			code: 'a { @my-at-rule { foo: 1 } }',
+		},
+		{
+			code: 'a { @my-other-at-rule { foo: 1 } }',
+		},
+	],
+
+	reject: [
+		{
+			code: 'a { @import { foo: 1; } }',
+			message: messages.rejected('foo'),
+			line: 1,
+			column: 15,
+		},
+		{
+			code: 'a { @my_at_rule { foo: 1; } }',
+			message: messages.rejected('foo'),
+			line: 1,
+			column: 19,
+		},
+		{
+			code: 'a { foo: 1; }',
+			message: messages.rejected('foo'),
+			line: 1,
+			column: 5,
+		},
+	],
+});
+
+testRule({
```

Comment:
[question] What reason to split this `testRule` from the `testRule` above?

Reply:
I took the tests from max-nesting-depth and modified them for my needs. The first test uses 2 strings (the second being regex inside a string) while the second tests uses regex directly. I'd be more than happy to merge if you feel it's unnecessary.

Reply:
I got it. I feel it more understandable if the two `testRule` are merged to one `testRule` like this:

```js
config: [true, { ignoreAtRules: ['media', '/^my-/', /^custom-/i] }],
```

But this is just my opinion. You can ignore it. 👍

Reply:
Also, it seems better to add a test case for an at-rule with arguments, for example:

```js
code: 'a { @my-at-rule(foo: bar) { foo: 1; } }',
```

Reply:
Will do!